prow/tide/tide.go

```diff
@@ -246,11 +247,21 @@ func requirementDiff(pr *PullRequest, q *config.TideQuery) (string, int) {
 			desc = fmt.Sprintf(" Should not have %s labels.", strings.Join(trunced, ", "))
 		}
 	}
+	diff = len(missingLabels) + len(presentLabels)
+
+	if desc == "" {
+		for _, excludedBranch := range q.ExcludedBranches {
+			if string(pr.BaseRef.Name) == excludedBranch {
+				desc = fmt.Sprintf(" Merging to branch %s is forbidden.", pr.BaseRef.Name)
+			}
+		}
+		diff = 1
```

Comment:
Isn't this really one more diff vs a single diff?

Reply:
Looks like this return value is consumed by `expectedStatus()`. Can we document the meaning of this return value?

Reply:
I branched this out so we only consider it when other diffs don't apply, so no in this case there are no other diffs and it is just `1`. This means that if there's a query against which everything passes except for the branch issue, it will be prioritized. Unclear if that is the right direction /cc @cjwagner

Reply:
We should check that the branch is acceptable first instead of last. If it is unacceptable we should prioritize that message so that users don't try to meet the other merge requirements only to discover that PRs can't merge to that branch at all.

Reply:
Any appetite for documenting the meaning of diff count?

Reply:
You know me, always ravenous for documentation

:hamburger: :fries: :beer: